bundles/org.eclipse.ui/schema/perspectiveExtensions.exsd

```diff
@@ -88,6 +89,28 @@ are appended to the initial definition.
                </restriction>
             </simpleType>
          </attribute>
+         <attribute name="editorOnboardingText" type="string">
+            <annotation>
+               <documentation>
+                  The text is shown in the editor area in case no editor is open.
+               </documentation>
+               <appInfo>
+                  <meta.attribute translatable="true"/>
+               </appInfo>
+            </annotation>
+         </attribute>
+         <attribute name="editorOnboardingImage" type="string">
+            <annotation>
+               <documentation>
+                  The image is shown in the editor area in case no editor is open.
+The image shall be grey and not colored and shall have a size of 350 x 350 px.
+Plus a second image for high resolution with a size of 700 x 700 px and a name like [image_name]@2x.[image_type].
+               </documentation>
+               <appInfo>
+                  <meta.attribute kind="resource"/>
```

Comment:
why is this rendered as text field in the plugin.xml editor. It does not have a value help.
Compare e.g. to the org.eclipse.ui.views extensions point (from the same plugin). Here for the "icon" attribute you have a browse button and the label contains a link to navigate. We should have the same here.

Reply:
done. require-bundle for org.eclipse.ui was missing in the example plugin.